NitroxClient/Communication/Packets/Processors/InitialPlayerSyncProcessor.cs

```diff
@@ -194,13 +196,19 @@ private void SetPlayerGuid(string playerguid)
             Log.Info("Received initial sync Player Guid: " + playerguid);
         }    
         
-        private void SpawnPlayerEquipment(List<EquippedItemData> equippedItems)
+        private void SpawnPlayerEquipment(List<EquippedItemData> equippedItems, bool vehiclesToSpawn)
         {
             Log.Info("Received initial sync packet with " + equippedItems.Count + " equipment items");
 
-            using (packetSender.Suppress<EquipmentAddItem>())
+            EquipmentItemAdder itemAdder = new EquipmentItemAdder(packetSender, equippedItems);
+
+            if (vehiclesToSpawn)
```

Comment:
This should always be based on base pieces to spawn as we are listening on an even that would never fire if there were no base pieces.